rfcs/text/0001-rfc-process.md

```diff
@@ -42,20 +48,28 @@ To address these needs, the DCP RFC process defines a transparent and standard m
 
 ### When is an RFC required?
 
-- An RFC is required for **substantial** additions, deletions, or changes to system behavior or semantics (API, major features, data model, protocols,​ ​service guarantees, architecture). In [semver](https://semver.org/) terms, *major* changes require an RFC, while *minor* changes are often candidates (new API addition).
+- An RFC is REQUIRED for **substantial** additions, deletions, or changes to system behavior or semantics (API, major features, data model, protocols,​ ​service guarantees, architecture). In [semver](https://semver.org/) terms, *major* changes require an RFC, while *minor* changes are often candidates (new API addition).
 
     **NOTE**: *Internal-only implementation decisions, bug fixes, refactoring, and performance optimization are not substantial changes and can continue to be documented and tracked using github issues.*
 
-- An RFC is required if a design impacts multiple DCP projects.
+- An RFC is REQUIRED if a design impacts multiple DCP projects.
 
-- An RFC is required for all changes to the DCP formal governance, including but not limited to oversight, decision-making, conflict resolution, and community processes such as charters and RFCs. For more background on governance, see the **Model C: Delegated Governance** section in [Organization & Structure of Open Source Software Development Initiatives](https://dash.harvard.edu/bitstream/handle/1/30805146/2017-03-24_governance.pdf).  
+- An RFC is REQUIRED for all changes to the DCP formal governance, including but not limited to oversight, decision-making, conflict resolution, and community processes such as charters and RFCs. For more background on governance, see the **Model C: Delegated Governance** section in [Organization & Structure of Open Source Software Development Initiatives](https://dash.harvard.edu/bitstream/handle/1/30805146/2017-03-24_governance.pdf).  
 
 ### Revising RFCs
 
-Where necessary, existing RFCs can be revised using the same process (a proposed change submitted as a RFC pull request and subject to community review and approval). **Major changes should result in a new RFC.**
+Where necessary, existing RFCs MAY be revised using the same process (a proposed change submitted as a RFC pull request and subject to community review and approval). **Major changes SHOULD result in a new RFC.**
+
+Minor and trivial changes MAY be made freely, with no announcement by the author(s) of the RFCs. Such changes include correcting typos, re-wordings, and the like.
+
+#### Deprecated RFCs 
```

Comment:
Editorial: just clarifying - is this section considered to be a sub-section of **Revising RFCs**?